config/locales/en.yml

```diff
@@ -1112,8 +1112,8 @@ en:
     new_case_button: Create case
     show_search_result_order_by_newest_first: Show newest cases first
     show_search_result_order_by_oldest_first: Show oldest cases first
-    show_search_result_order_by_newest_destruction_date: Show newest destruction date first
-    show_search_result_order_by_oldest_destruction_date: Show oldest destruction date first
+    show_search_result_order_by_newest_destruction_date: Show oldest destruction date first
```

Comment:
Wouldn't be better to leave the translation as before and just change where the translations are being used instead? As now it is confusing as the key says `order_by_newest` and the translation says `oldest first` 🤣 

Also not consistent with existing translations like `show_search_result_order_by_newest_first` and `show_search_result_order_by_oldest_first`

Reply:
Yes that would be better 😂 I'll do that